Package.swift

```diff
@@ -11,8 +11,8 @@ let package = Package(
         .library(name: "MobiusTest", targets: ["MobiusTest"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/Quick/Nimble", .upToNextMajor(from: "7.0.0")),
-        .package(url: "https://github.com/Quick/Quick", .upToNextMinor(from: "1.2.0")),
+        .package(url: "https://github.com/Quick/Nimble", from: "8.0.0"),
+        .package(url: "https://github.com/Quick/Quick", from: "2.0.0"),
```

Comment:
I believe this should be `from: "2.1.0"`, since Carthage users are free to start depending on hypothetical 2.1 features

Reply:
Updated